sdk/communication/azure-communication-chat/README.md

```diff
@@ -174,7 +174,7 @@ chatThreadClient.updateTopic("New Topic");
 
 #### Send a chat message
 
-Use the `sendMessage` method to sends a chat message to a chat thread identified by chatThreadId.
+Use the `sendMessage` method to sends a chat message to the chat thread that the `chatThreadClient` was created with.
```

Comment:
Use the `sendMessage` method to sends =>  
Use the `sendMessage` method to send